README.md

```diff
@@ -13,18 +13,23 @@
 [![GoDoc](https://godoc.org/github.com/getsentry/sentry-go?status.svg)](https://godoc.org/github.com/getsentry/sentry-go)
 [![go.dev](https://img.shields.io/badge/go.dev-pkg-007d9c.svg?style=flat)](https://pkg.go.dev/github.com/getsentry/sentry-go)
 
-`sentry-go` provides a Sentry client implementation for the Go programming language. This is the next line of the Go SDK for [Sentry](https://sentry.io/), intended to replace the `raven-go` package.
+`sentry-go` provides a Sentry client implementation for the Go programming
+language. This is the next line of the Go SDK for [Sentry](https://sentry.io/),
+intended to replace the `raven-go` package.
 
 > Looking for the old `raven-go` SDK documentation? See the Legacy client section [here](https://docs.sentry.io/clients/go/).
 > If you want to start using sentry-go instead, check out the [migration guide](https://docs.sentry.io/platforms/go/migration/).
 
 ## Requirements
 
-We verify this package against N-2 recent versions of Go compiler. As of September 2019, those versions are:
+The only requirement is a Go compiler.
 
-* 1.11
-* 1.12
-* 1.13
+We verify this package against the 3 most recent releases of Go. Those are the
+supported versions. The exact versions are defined in
+[`.travis.yml`](.travis.yml).
```

Comment:
This line doesn't have to be wrapped.

Reply:
The wrapping is only for reading the file without rendering. GitHub renders from markdown as a single paragraph that wraps according to screen size, this is the rendered output https://github.com/rhcarvalho/sentry-go/blob/readme/README.md